src/games/Lexible/models/ClientModel.ts

```diff
@@ -18,6 +18,15 @@ export const getLexibleClientTypeHelper = (
  {
      return {
         rootTypeName: "LexibleClientModel",
+        getTypeName(o) {
```

Comment:
Nice.  This is simple and keeps all relavate bits in the same place.  Also easy to discover when we forget to put a type there.